sdcm/scylla_bench_thread.py

```diff
@@ -183,7 +185,9 @@ def create_stress_cmd(self, stress_cmd, loader, cmd_runner):
                     # server_names = ' '.join(f'-tls-server-name {ip}' for ip in ips.split(","))
                     # stress_cmd = f'{stress_cmd.strip()} -tls-host-verification {server_names}'
 
-        return stress_cmd
+        if is_shell_supported:
+            return stress_cmd
+        return stress_cmd.removeprefix('scylla-bench')
```

Comment:
There is logic that depends on the tool/binary name that is going to be used.
For example, dynamic loader in K8S uses it.
Maybe somewhere else.

Reply:
Fixed that part of logic to use env instead